client/src/compontents/register/Register.js

```diff
@@ -1,11 +1,48 @@
-import React,{useState} from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Form from './../../theme/form';
+import Label from './../../theme/label';
+import Input from './../../theme/input';
+import Button from './../../theme/button';
+
+
+const P = styled.p`
+        margin-left:25%;
+        font-size:12px;   
+        &:hover{
+        text-decoration-color:gray;
+        text-decoration:underline;
+        cursor:pointer;
+}
+`
+
+const H2 = styled.h2`
+    text-align:center;`
+
+const FormWrap = styled.div`
+     margin-top:30px;
+    margin-left:25%;
+    `
+
+function Register(props) {
 
-function Register() {
     return (
-        <div>
-            Register Form
-        </div>
+
+        <Form form="register">
+            <H2>Create an Account </H2>
+            <FormWrap >
+                <Label htmlFor="name" >Username:</Label><br />
+                <Input type="text" name="name" id="name" required></Input><br />
```

Comment:
You could probably clean up those labels and inputs by combing them like so:
```suggestion
<label>Username: <br />
   <input type="text" name="name" id="name" required />
</label>
```
I think you can close off that input by with the forward slash in the end like so `<input ..... />`

Reply:
I think it is all based on people's preference. but I can change that as you suggested